e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -347,7 +347,7 @@
             "version": "1.0",
             "type": "docker",
             "status": "running",
-            "restartPolicy": "always",
+            "restartPolicy": "never",
```

Comment:
Was this change intentional?

Reply:
Yes it isn't designed to restart and still function. I encountered some confusing behavior when debugging this so was trying to clean this up.